src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -23,11 +23,17 @@
 
 			<!-- TODO: Alter CTA if Checkout is ready -->
 			<campaign-hero
+				v-show="pageData.page.contentGroups.mlCampaignHero"
```

Comment:
Did you try to use `v-if` for these two visibility checks?

Also, it will be more safe to either check a direct value (maybe type) on `heroAreaContent` or create a computed property that returns a boolean so you can use a v-if/v-else on these two components.

Reply:
The optional chaining within the js below is def the way to go with contentful based data. Ui doesn't have things setup to allow that in the template though.

Reply:
done